app/Http/Resources/PollResource.php

```diff
@@ -17,8 +19,11 @@ public function toArray($request)
         return [
             'id' => $this->id,
             'name' => $this->name,
-            'url' => $this->url,
-            'users' => UserResource::collection($this->whenLoaded('users')),
+            'is_admin' => count($this->users) > 0 ? $this->users->find(Auth::id())->pivot->admin : 0,
+            'total_user_count' => count($this->users),
+            'user_count' => count(DB::select('select user_id from poll_ratings where poll_id = ? GROUP BY user_id', [$this->id])),
```

Comment:
Isn't it possible to use a select count(*)?

Reply:
Maybe but i couldn't figure how to get the output of the select query in usable form.